src/config/keys.ts

```diff
@@ -51,7 +51,7 @@ export const itemKeys = {
       allChildren,
 
       // itemKeys.single(id).children([one, two])
-      children: (types?: UnionOfConst<typeof ItemType>[]) =>
+      children: (types: UnionOfConst<typeof ItemType>[] = []) =>
```

Comment:
Mhmm ?? you remove the possibility for it to be optional but you provide a default value ?

Reply:
Yes, it is to avoid having two keys that represent the same thing. Without this consideration, in the frontend, I end up with:

- `['items', 'one', :id, 'children', undefined]` when the filter is not used.
- `['items', 'one', :id, 'children',[]]` when the filter is used, but no type is provided for filtering.

This is why I made the change here, to avoid handling this in the frontend.

Reply:
Okay makes sens, sorry I got confused by the removal of the `?` but indeed you do not need the `?` if you provide an initialiser. My bad !